### ISSUE TITLE
Render training back to video

### ISSUE THREAD

**Issue by heimrabe** [coltonstearns/dynamic-gaussian-marbles#10]
Hi thank you very much for your work. I am wondering, is it possible to render the training back to the resolution and size of the original video? I see that the rendering is done with a nerfstudio function which adds the tracks on the reconstruction.  I am trying to train on some custom real world videos and would like to get a video which is a reconstruction of the real video from the training. Is in the model the outputs["rgb"] the rendering of each frame?

**Comment by coltonstearns** [comment]
Yes, just comment out line 36 of `src/visualization/render.py` (i.e. the line with `cameras.metadata['viz_tracking'] = torch.ones_like(cameras.times) * 0`). This will turn off the point-track rendering. I can also add this toggle as an option in a future code push. Let me know if you have any more questions about this!
